content/en/load-testing/Manual/800-master-controller-cheatsheet.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Mastercontroller Cheat Sheet"
+linkTitle: "Cheat Sheet Mastercontroller"
+
+weight: 800
+type: docs
+
+description: >
+  You will find the usage instructions and a list of available commands below.
```

Comment:
Maybe a short summary of the purpose of the master controller would be nice. Maybe something like "primary tool to run the load tests", "can be used interactive or in auto mode"... something like that...